crates/forge/tests/e2e/running.rs

```diff
@@ -997,3 +997,30 @@ fn incompatible_snforge_std_version_warning() {
         "}
     );
 }
+
+#[test]
+fn detailed_resources_flag() {
+    let temp = setup_package("erc20_package");
```

Comment:
I would get rid of this test, this will just make us update mindlessly those numbers when something changes and it doesn't really change anything other than stdout format

Reply:
The numbers could be omitted
```                
steps: [..]
memory holes: [..]
builtins: ([..])
syscalls: ([..])
```

Reply:
please do that if you'd like to keep that one

Reply:
We need to keep this one imo - it checks the e2e functionality (if we print the resources at all). Agree with omitting the numbers though